itou/www/signup/tests/test_job_seeker.py

```diff
@@ -140,12 +144,15 @@ def test_job_seeker_temporary_nir(self):
         url = reverse("signup:job_seeker")
         response = self.client.get(url)
         assert response.status_code == 200
+        # Since provided no NIR was provided, suggest nothing
```

Comment:
Ça doit être ça que tu voulais dire :P, et peut-être corriger le fait qu'un NIR a tout de même été donné mais qu'on ne l'utilise pas car temporaire
```suggestion
        # Since no NIR was provided, suggest nothing
```